config/default.json.template

```diff
@@ -61,6 +61,12 @@
     "newsletters": true,
     "log": true
   },
+  "project": {
+     "enable_group": true,
+     "default_size": 500,
+     "default_path": "/opt/project",
+     "default_expire": 360
```

Comment:
I suppose it is in days? Could you just add a commebt specifying it, ok for merge

Reply:
I think json does not support comment, where do you want the comment to be added ?

Reply:
correct... :-(

Config fields/sections should be explained somewhere

In a config/README.md?

Reply:
I have started it here https://github.com/genouest/genouestaccountmanager/pull/153
But for the moment I am not soo happy with readability of the final result